File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
a = int(input())
b = int(input())
c = int(input())

if a + b + c == 180:
  if a == b == c:
    print("Equilateral")
  elif a == b != c or a == c != b or b == c != a:
    print("Isosceles");
  else:
    print("Scalene")

else:
  print("Error")

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.c
#include <stdio.h>

int main(void) {  
   int a, b, c;
   scanf("%d", &a);
   scanf("%d", &b);
   scanf("%d", &c);

   if(a + b + c == 180){
      if(a == b && a == c){
         printf("Equilateral");
      }
      else if(a == b && a != c || a == c && a != b || b == c && b != a){
         printf("Isosceles");
      }
      else{
         printf("Scalene");
      }
   }
   else{
      printf("Error");
   }

   return 0;
}